### DISCUSSION TITLE
foreign IP address detected using "netstat -f" while farming Chia

### DISCUSSION THREAD

**Discussion by chenus1** [Chia-Network/chia-blockchain#5045] [Support]
I detected some IP addresses from Russia and BR are established using "netstat -f" while doing farming in main machine. Is there any risk concern? Please advise!

netstat -f
  TCP    main machine IP:53525     63.249.43.210.rhinocommunications.net:8444  ESTABLISHED
  TCP    main machine IP:53627     95-165-171-155.static.spd-mgts.ru:8444  ESTABLISHED
  TCP    main machine IP:53634     bd79e887.virtua.com.br:8444  ESTABLISHED
  TCP    main machine IP:53648     207-38-170-5.s8938.c3-0.43d-cbr1.qens-43d.ny.cable.rcncustomer.com:8444  ESTABLISHED
  TCP    main machine IP:53685     138.99.204.176.clientes.teleson.net.br:8444  ESTABLISHED
  TCP    main machine IP:53730     31.10.110.59:8444      ESTABLISHED
  TCP    main machine IP:53810     rrcs-24-173-215-254.sw.biz.rr.com:8444  ESTABLISHED
  TCP    main machine IP:53947     cpe-74-141-11-177.neo.res.rr.com:8444  ESTABLISH

**Comment by T4C-FanBoy** [comment]
They sync with your node, so its normal to see strange ip

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
